src/lib/util/validateKey.js

```diff
@@ -0,0 +1,6 @@
+export default function validateKey(object, key, errorMessage) {
```

Comment:
We are only using this in the `isHash` validator, I don't think we need a separate util for this